docs/pages/access-controls/reference.mdx

```diff
@@ -357,9 +357,39 @@ allow:
     - resources:
         - token
       verbs: [list, create, read, update, delete]
+```
+
+### Allowing access to token resources
 
+You should note that if you configure a role that allows tokens to be created, a user assigned to the 
+role can create any type of token. 
+For example, you might create a role with the following configuration to enable assigned
+users to enroll servers:
```

Comment:
I would phrase this in a way that describes the current behavior without sounding like a defect.

```suggestion
Users with permissions to create provision tokens can include any Teleport
system roles in the token.

For example, you might create a role with the following configuration to enable
users to enroll servers:
```

Reply:
I haven't seen "system role" or "provision token" terminology. Maybe:
Users with permission to create tokens can use that permission to provision any type of Teleport resource.

Reply:
`ProvisionToken` is the literal name of the resource that is created when you run `tctl tokens add`, though the docs seem to use the term "join token" more. I think either is fine.

There are other types of "tokens" like password reset tokens which are controlled separately in RBAC, so I think being explicit is helpful.


Similarly, a "system role" is app, node, proxy, auth, db, windowsdesktop, kube, etc. (The different modes a Teleport agent can operate in.) These are distinct from roles that you would assign to users (like access, auditor, editor, etc.)

Reply:
Thanks for the clarification. Might want to revisit the token terminology because I also see it called an invitation token in messages after you generate it.
Overloading "roles" terminology is a common (but not Teleport-specific) issue. I'd prefer to keep roles aligned with what you configure and assign for use access.

Reply:
Interesting. Typically "invitation token" is meant to invite new users (not services) to the cluster. This could definitely use some cleaning up!